### PR TITLE
add craned dashboard frontend for chart

### PR DESCRIPTION
**Pull request by chenkaiyue**

### CODE REVIEW

File: charts/crane/templates/craned-deployment.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ spec:
       serviceAccountName: craned
       containers:
         - name: craned
-          image: "{{ .Values.craned.image.repository }}:{{ .Values.craned.image.tag | default .Chart.AppVersion }}"
+          image: ccr.ccs.tencentyun.com/chenky/crane:022801
```
Comment (chenkaiyue) [inline charts/crane/templates/craned-deployment.yaml:30]: for test now

**Comment by chenkaiyue** [comment]
image
image
image

**Comment by chenkaiyue** [comment]
@qmhu @kitianFresh

File: charts/crane/values.yaml [hunk 2]
```diff
@@ -76,4 +76,10 @@ craneAgent:
 
   containerArgs:
     runtime-endpoint: unix:///rootvar/run/dockershim.sock
-    v: 2
\ No newline at end of file
+    v: 2
+
+cranedDashboard:
+  image:
+    fullImageName: docker.io/gocrane/dashboard:v0.2.0
```
Comment (qmhu) [inline charts/crane/values.yaml:83]: The image version will update pre release, so suggest to use repository.

**Pull request merged**